### DISCUSSION TITLE
What does Blink use the Camera for?

### DISCUSSION THREAD

**Discussion by egberts** [blinksh/blink#1551] [Q&A] [answered]
What is the camera permission in System->Privacy used for, as it wasn't listed in the Apple App Store, App Privacy section.

**Comment by pcho** [comment, marked as answer]
> What is the camera permission in System->Privacy used for, as it wasn't listed in the Apple App Store, App Privacy section.

It’s for `facecam` feature. You can see usage example here: https://www.youtube.com/watch?v=tPHjcsrKM9s

```
OVERVIEW: Show facecam on the screen

USAGE: facecam [--verbose] [--quiet] <subcommand>

OPTIONS:
  --verbose
  --quiet
  -h, --help              Show help information.

SUBCOMMANDS:
  on (default)            Turn on facecam
  off                     Turn off facecam

  See 'facecam help <subcommand>' for detailed help.
```

**Comment by carloscabanero** [comment]
Note that this camera is just for personal usage, and we are not sending any "videos" of you or anything like that there. It is part of the permissions we request, but I do not think it has to be mentioned as part of the privacy section.

There is also a "keycast" option that will show what you are typing as you go :)
